src/openloco/ui/dropdown.h

```diff
@@ -63,6 +63,7 @@ namespace openloco::ui::dropdown
     void set_highlighted_item(size_t index);
     void set_item_selected(size_t index);
 
+    void show(int16_t x, int16_t y, int16_t width, int16_t height, colour_t colour, size_t count, uint8_t itemHeight, uint8_t flags);
```

Comment:
This same signature is already defined on line 68, right?

Reply:
Yep, that snuck in there on a rebase from the construction PR I think